src/index.html

```diff
@@ -4,9 +4,391 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="logotip">
```

Comment:
```suggestion
      <img
        class="logo"
        src="images/logo.png"
        alt="logotip"
      >
```
check your code style